### PR TITLE
added prettier config, components, and sass

### PR DESCRIPTION
**Pull request by m-spilman**

### CODE REVIEW

**Review by ahaly92 — APPROVED** [review]

**Review by BarretteBanner — APPROVED** [review]

**Comment by m-spilman** [comment]
changed <a> to <div> cleaned up some names (camelCase). no more errors in console : ) 
let me know if you wanted this on a new branch - so further updates will be on new branches. And if once each pull gets approved if you want us squash? - Thanks for your help!

**Review by BarretteBanner — APPROVED** [review]
Looks great! This can be on this branch. It's once you merge that you should create a new branch. And you should always rebase / squash before a pr (and after updating a pr)  that way when you're told to ship it you can just press 'squash and merge'

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                className="nav-link disabled"
+                href="#"
+                tabIndex="-1"
+                aria-disabled="true"
+              >
+                Disabled
+              </div>
+            </li>
+            <li className="nav-item dropdown" onClick={toggle}>
+              <div
```
Comment (ahaly92) [inline src/components/Header/Header.js]: use react's Link component here instead of a div with an href.

File: src/components/Main/Main.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import Orders from '../../pages/Orders';
+import OrderSuccess from '../../pages/OrderSuccess';
+import Products from '../../pages/Products';
+import ShoppingCart from '../../pages/ShoppingCart';
+
+function Main() {
+  return (
+    <main className="main">
+      <Switch>
+        <Route exact path="/" components={Home}></Route>
```
Comment (BarretteBanner) [inline src/components/Main/Main.js]: Let's have your paths look like this: "/admin-orders"

File: src/components/Main/Main.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <Route exact path="/AdminOrders" components={AdminOrders}></Route>
+        <Route exact path="/AdminProducts" components={AdminProducts}></Route>
+        <Route exact path="/CheckOut" components={CheckOut}></Route>
+        <Route exact path="/Orders" components={Orders}></Route>
+        <Route exact path="/OrderSuccess" components={OrderSuccess}></Route>
```
Comment (BarretteBanner) [inline src/components/Main/Main.js]: Let's try adding a path that re-directs us to '/' if they type in something that doesn't match any of the options in the url. There are multiple ways to do this so when you find a solution you like reach out so we can discuss first before you implement it.

File: src/components/Main/Main.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      <Switch>
+        <Route exact path="/" component={Home} />
+        <Route exact path="/admin-orders" component={AdminOrders} />
+        <Route exact path="/admin-products" component={AdminProducts} />
+        <Route exact path="/check-out" component={CheckOut} />
+        <Route exact path="/orders" component={Orders} />
+        <Route exact path="/order-success" component={OrderSuccess} />
+        <Route exact path="/products" component={Products} />
+        <Route exact path="/shopping-cart" component={ShoppingCart} />
+        <Route path="/" component={Home} />
```
Comment (BarretteBanner) [inline src/components/Main/Main.js]: Let's make this one re-direct to the '/' and maybe change the '/' to '*'

**Review by BarretteBanner — APPROVED** [review]
Well done!

**Pull request merged**